src/hotspot/share/logging/logTagSet.hpp

```diff
@@ -110,6 +112,7 @@ class LogTagSet {
   // of its current outputs combined with the given decorators.
   void update_decorators(const LogDecorators& decorator = LogDecorators::None);
 
+  void label(outputStream* st, const char* separator = ",") const;
   int label(char *buf, size_t len, const char* separator = ",") const;
```

Comment:
This old label() is only used in LogTagset. You can make it private.

Further,  no one consumes its return value now.  you can change its rettypea void.  it will avoid different return types for overloaded member functions.

Reply:
I thought so too, but the gtests still test this interface. Seeing that its not used in the hs codebase anymore I could remove it, but would have to rewrite all gtests to use the new interface instead. I'd rather avoid this to keep the patch small. Lets do this in a different RFE.

Reply:
understood. fair enough.